Docs: Configuring external DNS for a custom domain (#1803)
vtex/faststore: apps/site/pages/docs/go-live/2-configuring-external-dns.mdx

<header>

> # 2. Configuring external DNS for a custom domain

Author: ```suggestion
# Part 2: Configuring external DNS for a custom domain
```

<Callout type="warning" emoji="⚠️">
This documentation is currently under development.
</Callout>

In this guide, you will learn how to configure external domains for your **FastStore + VTEX** website. The process involves the following steps:

>  - Set up your external DNS provider to point your custom domain to your VTEX website. Custom domains can be purchased from any **domain registrar**.

Reviewer: ```suggestion
 - Set up your external DNS provider to point your custom domain to your VTEX website. Custom domains can be purchased from any **domain registration platform**.
```

is that?

 - Configure your custom domains in **VTEX Account Settings**.
 - Associate your custom domains with your **FastStore** project. 

Besides making your FastStore website accessible via a custom domain, the **VTEX Checkout** will also be operational as soon as you complete this guide. You'll then be ready to continue configuring other VTEX solutions, such as **Order Placed**, **Login**, and **My Account**.

---

## Before you start

> - [Set up your **VTEX** account.](/docs/go-live/1-setting-up-vtex-account-for-faststore)

Reviewer: ```suggestion
- [Set up your VTEX account.](/docs/go-live/1-setting-up-vtex-account-for-faststore)
```

+  loginUrl: 'https://secure.mystore.com/api/io/login',
+  accountUrl: 'https://secure.mystore.com/api/io/account',

// Lighthouse CI
lighthouse: {
...
```
4. Save your changes.
5. Open the `vtex.env` file.
6. Add the `SITE_HOST` key with the main domain specified in [Configuring domains in VTEX Account Settings](/docs/go-live/2-configuring-external-dns#step-2---configuring-domains-in-vtex-account-management) as the value.
   
```diff title="vtex.env"
+ SITE_HOST=mystore.com
```

7. Save your changes.
8. Open a Pull Request, commit your changes, and deploy them to `main`/`master`.

### Step 4 - Configuring the CDN workflow (Only for new go-lives)

If your FastStore website is going live for the first time, [open a ticket with VTEX Support](https://help.vtex.com/en/support) and specify that you need to configure the CDN workflow for the secure and main domains of your store. Remember to include the following information in your ticket:

- The account **name**.
- The **secure** and **main** domains of your store.

The VTEX team will evaluate your request and take the necessary actions to configure the CDN workflow for your account.

---

> ## Related resources

Reviewer: ```suggestion
## Related Resources
```
